src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -942,6 +946,7 @@ namespace OpenMS
       if (pep->isPeptide())
       {
         pep_hit_.setSequence(AASequence::fromString(pep->sequence));
+        mrmfeature->setMetaValue("missedCleavages", pd.peptideCount(pep_hit_.getSequence()));
```

Comment:
I dont think this is a great idea, since this basically only measures the library cleavages -- actually I was planning to remove the peptide hits from here.

Reply:
yeah sure, one can only see MCs if the library contains them. But OpenSWATH is targeted after all, so I don't see a way around that. And if the library contains the MC's then there should be no problem.

Reply:
as we discussed, I think we would need a more targeted approach to detect MC, e.g. if you have a protein 
PEPTIDEKPEPAKPEPBK

then we would have to compare intensities of 
- PEPTIDEK
- PEPAK
- PEPBK
- PEPTIDEKPEPAK
- PEPAKPEPBK
- PEPTIDEKPEPAKPEPBK

across batches and get some information on trypsin efficiency and I think that would be more effective.